src/Application/Regulation/Command/SaveRegulationLocationCommandHandler.php

```diff
@@ -63,6 +80,17 @@ public function __invoke(SaveRegulationLocationCommand $command): Location
         foreach ($command->measures as $measureCommand) {
             if ($measureCommand->measure) {
                 $measuresStillPresentUuids[] = $measureCommand->measure->getUuid();
+
+                foreach ($measureCommand->measure->getLocations() as $location) {
+                    $location->update(
+                        cityCode: $command->cityCode,
+                        cityLabel: $command->cityLabel,
+                        roadName: $command->roadName,
+                        fromHouseNumber: $command->fromHouseNumber,
+                        toHouseNumber: $command->toHouseNumber,
+                        geometry: $geometry,
+                    );
+                }
```

Comment:
Cette partie est un peu bizarre, car on ne voudra jamais recopier les données d'une ancienne location dans _toutes_ les locations d'une mesure

Actuellement la mesure n'aura qu'_une seule_ `locationNew`. Mieux vaut l'assumer ?

```suggestion
                $measureCommand->measure->getLocationNew()->update(
                    cityCode: $command->cityCode,
                    cityLabel: $command->cityLabel,
                    roadName: $command->roadName,
                    fromHouseNumber: $command->fromHouseNumber,
                    toHouseNumber: $command->toHouseNumber,
                    geometry: $geometry,
                );
```

Reply:
Done